README.md

```diff
@@ -19,6 +19,25 @@ A modern, clean, and very simple responsive HTML invoice template, because somet
 ## How to use
 Open the [invoice.html](https://github.com/sparksuite/simple-html-invoice-template/blob/master/invoice.html) file and use its HTML and CSS in your application.
 
+## Printer-friendly styling
+By default, the layout of the invoice when printed is equal to the regular layout. Optionally, the box-shadow and border line can be removed when printing, to give a more invoice-like feeling.
+<p align="center">
+  <img alt="Without printer friendly styling" src="https://user-images.githubusercontent.com/22718003/147985007-fb298dd8-b88c-4407-bdea-26fe2c9f595c.png" width="45%">
+&nbsp; &nbsp; &nbsp; &nbsp;
+  <img alt="With printer-friendly styling" src="https://user-images.githubusercontent.com/22718003/147985049-ebf70654-ace3-4e49-b7e1-9fb4d3583163.png" width="45%">
+</p>
+
+Example:
```

Comment:
I'm a bit picky when it comes to verbiage. I'd reword this and remove the screenshot to reduce clutter:

```suggestion
By default, the appearance of the invoice when printed is the same as when viewed on a screen. If the invoice is the primary or only element on the page, you may want to consider removing the box-shadow and border, and letting the invoice extend the full width of the page. That can be accomplished with this CSS:
```